hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueService.kt

```diff
@@ -176,7 +176,27 @@ fun SqsAsyncClient.countAllMessagesOnQueue(queueUrl: String): CompletableFuture<
     }
 
 fun PublishRequest.Builder.eventTypeMessageAttributes(eventType: String): PublishRequest.Builder =
-  messageAttributes(mapOf("eventType" to SnsMessageAttributeValue.builder().dataType("String").stringValue(eventType).build()))
+  messageAttributes(mapOf(eventTypeSnsPair(eventType)))
+fun PublishRequest.Builder.eventTypeMessageAttributesNoTracing(eventType: String): PublishRequest.Builder =
+  messageAttributes(
+    mapOf(
+      eventTypeSnsPair(eventType),
+      "noTracing" to SnsMessageAttributeValue.builder().dataType("String").stringValue("true").build(),
```

Comment:
I suppose it would have been slightly better if this and the other attribute was namespaced; e.g 'uk.gov.hmpps.sqs.noTracing' but I suppose the risk is tiny it would clash with anything

Reply:
agreed, I did think that when creating it but then thought that it would just add complexity